src/Servant/Pagination.hs

```diff
@@ -109,8 +109,8 @@ module Servant.Pagination
   , defaultOptions
 
   -- * Use Ranges
-  , extractRange
-  , putRange
+  , ExtractRange(..)
+  , PutRange(..)
```

Comment:
Why this change exactly :thinking: ?
These internals were not exposed on purpose.